### PR TITLE
Docs: Add docs for log line permalinking

### PR DESCRIPTION
**Pull request by svennergr**
**What is this feature?**
This adds documentation for the permalinking of log lines feature developed in https://github.com/grafana/grafana/pull/69464

### CODE REVIEW

File: docs/sources/explore/logs-integration.md [hunk 1]
```diff
@@ -125,6 +125,10 @@ You may encounter long lines of text that make it difficult to read and analyze
 
 The **Open in split view** button allows you to execute the context query for a log entry in a split screen in the Explore view. Clicking this button will open a new Explore pane with the context query displayed alongside the log entry, making it easier to analyze and understand the surrounding context.
 
+## Copy link to log line
+
+Linking of log lines in Grafana allows you to quickly navigate to specific log entries for precise analysis. By clicking on the 🔗 button of a log line, you can generate and copy a [short URL]({{< relref "../developers/http_api/short_url/" >}}) that provides direct access to the exact log entry within an absolute time range. When you open the link, Grafana will automatically scroll to the corresponding log line and highlight it with a blue background, making it easy to identify and focus on the relevant information.
```
Comment (JStickler) [inline docs/sources/explore/logs-integration.md:130]: Is there a name/hover text for this button for screen readers?

Comment (svennergr) [reply docs/sources/explore/logs-integration.md:130]: There is, it's called `Copy shortlink`:

image

Comment (JStickler) [inline docs/sources/explore/logs-integration.md:130]:
```suggestion
Linking of log lines in Grafana allows you to quickly navigate to specific log entries for precise analysis. By clicking the *Copy shortcut* 🔗 button for a log line, you can generate and copy a [short URL]({{< relref "../developers/http_api/short_url/" >}}) that provides direct access to the exact log entry within an absolute time range. When you open the link, Grafana will automatically scroll to the corresponding log line and highlight it with a blue background, making it easy to identify and focus on the relevant information.
```

**Review by JStickler — APPROVED** [review]
[docs team] LGTM

**Pull request merged**